packages/features/webhooks/lib/factory/BookingWebhookServiceFactory.ts

```diff
@@ -0,0 +1,44 @@
+import { WebhookRepository } from "../repository/WebhookRepository";
+import { BookingWebhookService } from "../service/BookingWebhookService";
+import { WebhookNotificationHandler } from "../service/WebhookNotificationHandler";
+import { WebhookNotifier } from "../service/WebhookNotifier";
+import { WebhookService } from "../service/WebhookService";
+import { BookingPayloadBuilder } from "./BookingPayloadBuilder";
+import { FormPayloadBuilder } from "./FormPayloadBuilder";
+import { InstantMeetingBuilder } from "./InstantMeetingBuilder";
+import { MeetingPayloadBuilder } from "./MeetingPayloadBuilder";
+import { OOOPayloadBuilder } from "./OOOPayloadBuilder";
+import { RecordingPayloadBuilder } from "./RecordingPayloadBuilder";
+import { WebhookPayloadFactory } from "./WebhookPayloadFactory";
+
+export function createBookingWebhookService(): BookingWebhookService {
+  // Create payload builders
+  const bookingPayloadBuilder = new BookingPayloadBuilder();
+  const formPayloadBuilder = new FormPayloadBuilder();
+  const oooPayloadBuilder = new OOOPayloadBuilder();
+  const recordingPayloadBuilder = new RecordingPayloadBuilder();
+  const meetingPayloadBuilder = new MeetingPayloadBuilder();
+  const instantMeetingBuilder = new InstantMeetingBuilder();
+
```

Comment:
Shouldn't these instances be passed as dependencies here? Would require us to convert this _createBookingWebhookService_ into a class which would have these deps injected.

Reply:
This is actually dead code, it is best to remove this as we're not going to use this (at least not planned yet). This was left during cleanup when we moved to DI